presto-docs/src/main/sphinx/security/ldap.rst

```diff
@@ -314,3 +314,22 @@ with the appropriate :abbr:`SAN (Subject Alternative Name)` added.
 Adding a SAN to this certificate is required in cases where ``https://`` uses IP address in the URL rather
 than the domain contained in the coordinator's certificate, and the certificate does not contain the
 :abbr:`SAN (Subject Alternative Name)` parameter with the matching IP address as an alternative attribute.
+
+Authentication or SSL errors with JDK Upgrade
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Starting JDK 8u181 release, to improve the robustness of LDAPS (secure LDAP over TLS) connections, endpoint identification
```

Comment:
> Starting with the JDK 8u181 release, ...

Reply:
Please wrap these lines to ~80 characters. If they wrap on GitHub, they are too long